doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -699,6 +699,83 @@ If you need to remove the Tarantool Kubernetes operator, execute:
       persistentvolumeclaim "www-routers-1-0" deleted
       persistentvolumeclaim "www-storages-0-0" deleted
 
+
+.. _cartridge_kubernetes_cluster_management_failover:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Failover
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Failover - replicaset leader selection mechanism. You can read more about it `here <https://www.tarantool.io/en/doc/latest/book/cartridge/topics/failover/>`_.
+
+.. NOTE::
+
+   The ability to configure failover via kubernetes manifests will appear later
+
+********************************************************************************
+Eventual mode
+********************************************************************************
+
+Default mode. Uses SWIM protocol to detect failures.
+
+********************************************************************************
+Stateful mode
+********************************************************************************
+
+Uses external storage for coordination. To work, you need to enable a ``failover-coordinator`` role on several instances.
+
+To do this, add the role in *values.yml* to the description of the replicasets:
+
+.. code-block:: yaml
+
+  RoleConfig:
```

Comment:
Not enough indentation. Both the `..  code-block` directive and the code example [should be indented by 4 spaces](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/#wrapping-text-and-indentation). Here code example starts on 2 spaces, directive on 3 — this just won't build.